lib/actions-oclif/env/rm.ts

```diff
@@ -89,23 +89,23 @@ export default class EnvRmCmd extends Command {
 			);
 		}
 
-		return patterns
-			.confirm(
+		try {
+			await patterns.confirm(
 				options.yes || false,
 				'Are you sure you want to delete the environment variable?',
-			)
-			.then(function() {
-				if (options.device) {
-					return balena.pine.delete({
-						resource: 'device_environment_variable',
-						id: params.id,
-					});
-				} else {
-					return balena.pine.delete({
-						resource: 'application_environment_variable',
-						id: params.id,
-					});
-				}
-			});
+			);
+		} catch (err) {
+			if (err.message === 'Aborted') {
+				return patterns.exitWithExpectedError(err);
```

Comment:
If the user answers "No" to the question, call `exitWithExpectedError` to avoid Raven/Sentry error reporting (it's not a reportable error to answer "No" to a question...).